index.html

```diff
@@ -7,22 +7,25 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
     <link rel="stylesheet" href="./style.css" />
     <link rel="stylesheet" href="./app.css">
-    <title>Setup Mobile First Portfolio</title>
+    <title>Setup Portfolio Desktop Version</title>
 </head>
 <body>
     <main id="root">
-        <header class="navbar">
+    <header>
+        <nav class="navbar">
             <div id="logo">
-                 <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
-             </div>
-             <nav>
-                   <div id="img">
-                       <img class="hamburger" src="./images/Union.png" alt="">
-                   </div>           
-               </nav>
-        </header>   
-        <section id="index">
-            <div id="container-def">
+                <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
+            </div>  
+            <ul class="nav-items">
+                <li class="portfolio item"><a href="#works-container" class="link">Portfolio</a></li>
+                <li class="about item"><a href="#about-container" class="link">About</a></li>
+                <li class="contact item"><a href="#contact-container" class="link">Contact</a></li>
+                <li class="closeMenu"><i class="fa fa-times"></i></li>
+            </ul>
+        </nav>
+    </header>
```

Comment:
well done on implementing the desktop design for the header 👍, but we need to keep the hamburger icon and hide it in desktop size 

![hamberger icon missing](https://user-images.githubusercontent.com/59707859/221054178-3d10cbac-ea72-42bc-9d2d-79225f5da0fc.PNG)